packages/desktop-client/src/components/reports/DateRange.tsx

```diff
@@ -62,6 +62,12 @@ export function DateRange({ start, end, type }: DateRangeProps): ReactElement {
           : d.format(endDate, 'MMM yyyy')}
       </div>
     );
+  } else if (['budget', 'average'].includes(type || '')) {
+    content = (
+      <div>
+        Compare {d.format(startDate, 'MMM yyyy')} to {type}
```

Comment:
When "budgeted" button is chosen all other indicators say "budgeted" but this ones leaves out the "ed" and only says "budget"